src/index.html

```diff
@@ -8,7 +8,74 @@
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <a href="#home" class="logo__link">
```

Comment:
```suggestion
    <header class="header">
      <a href="#home" class="logo__link">
```
Don't leave empty lines between parent and child elements

Reply:
Thanks, I'll keep that in mind